src/areaCalculator/Main.java

```diff
@@ -3,6 +3,8 @@
 public class Main {
 
     public static void main(String[] args) {
-
+        AreaCalculator myCalculator = new AreaCalculator();
```

Comment:
Variable could be named simply calculator, prefix "my" does not add any real value here. 
Anyway, it is optional, you can fix it with other PR if you want.